docs: refer to the app as the Mend Renovate app (#23336)
renovatebot/renovate: docs/usage/configuration-options.md

## ignorePrAuthor

> This is usually needed if someone needs to migrate bot accounts, including from hosted app to self-hosted. // DRAFT: Do I need to change this line?

Reviewer: ```suggestion
This is usually needed if someone needs to migrate bot accounts, including from the Mend Renovate App to self-hosted.
```

## platformCommit

> Only use this option if you run Renovate as a [GitHub App](https://docs.github.com/en/developers/apps/getting-started-with-apps/about-apps). // DRAFT: Should I change this line?

Reviewer: ```suggestion
Only use this option if you run Renovate as a [GitHub App](https://docs.github.com/en/developers/apps/getting-started-with-apps/about-apps).
```

It does not apply when you use a Personal Access Token as credential.

When `platformCommit` is enabled, Renovate will create commits with GitHub's API instead of using `git` directly.

> This way Renovate can use GitHub's [Commit signing support for bots and other GitHub Apps](https://github.blog/2019-08-15-commit-signing-support-for-bots-and-other-github-apps/) feature. // DRAFT: Should I change this line?

Reviewer: ```suggestion
This way Renovate can use GitHub's [Commit signing support for bots and other GitHub Apps](https://github.blog/2019-08-15-commit-signing-support-for-bots-and-other-github-apps/) feature.
```

1. Enable the "Dependency graph"
1. Enable "Dependabot alerts"
1. If you're running Renovate in app mode: make sure the app has `read` permissions for "Vulnerability alerts".

>    If you're the account administrator, browse to the app (for example [https://github.com/apps/renovate](https://github.com/apps/renovate)), select "Configure", and then scroll down to the "Permissions" section and make sure that `read` access to "vulnerability alerts" is mentioned // DRAFT: Should I change this line?

Reviewer: ```suggestion
   If you're the account administrator, browse to the app (for example [the Mend Renovate App](https://github.com/apps/renovate)), select "Configure", and then scroll down to the "Permissions" section and make sure that `read` access to "vulnerability alerts" is mentioned
```
